apps/base-docs/docs/building-with-base-and-foundry/introduction.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Introduction to Foundry
+description: Documentation for Foundry, a toolchain for smart contract development. Provides instructions on setting up your development environment to work with Foundry.
+keywords: [Foundry, Forge, Foundry Book, smart contract development, toolchain]
+hide_table_of_contents: false
+---
+
+In this article, you'll learn how to setup Foundry, the toolchain for smart contract development to develop with Base.
+
+---
+
+## Objectives
+
+By the end of this lesson, you should be able to:
+
+- Install Foundry
+- Create my first Foundry project
+- Compile your first smart contract using Foundry
+- Configure Foundry to work with Base
+
+## Overview
+
+Foundry is a smart contract development toolchain that is composed of multiple small command line tools:
+
+- *[forge]*: Used to compile, tests, and deploy your smart contracts.
+- *[cast]*: Useful to interact with the Blockchain over RPC. You can make smart contract calls, send transactions, or retrieve any type of chain data.
+- *[chisel]*: Chisel is a solidity REPL. You can write solidity code directly.
+- *[anvil]*: A local Blockchain node for testing and development purposes.
+
+Using Foundry you can manage your dependencies, compile your project, run tests, deploy smart contracts and interact with the chain from the command-line and via Solidity scripts.
+
+For a deep dive on the Foundry features and full capabilities, check out the [Foundry Book].
+
+## Installing Foundry
+
+In order to install Foundry, you can use `Foundryup`, the Foundry's toolchain installer.
+
+To install `Foundryup` you have to run in the terminal:
+
+```bash
+$ curl -L https://foundry.paradigm.xyz | bash
+```
+
+After `Foundryup` is installed, you can now install `Foundry` by running:
+
+```bash
+$ foundryup
+```
+
+You can verify the installation by trying the following commands:
+
+```bash
+$ forge --version
+$ cast --version 
+$ chisel --version
+$ anvil --version
+```
+
+## My First Foundry Project
+
+To create a foundry project we can simply run:
+
+```bash
+$ forge init hello_foundry_in_base
+```
+
+This will create a foundry project with the following structure:
+
+```bash
+├── lib          # all the libraries installed will live in this folder
+├── script       # scripts folder, i.e deploy scripts
+├── src          # smart contracts folder
+├── test         # tests folder
+└── foundry.toml # foundry configuration file
+```
+
+You will also notice a `.gitsubmodules` file, this is because `Foundry` handles dependencies using [Git submodules].
```

Comment:
```suggestion
You will also notice a `.gitsubmodules` file -- this is because `Foundry` handles dependencies using [Git submodules].
```

Reply:
Are you missing a URL again? Or is this deliberate? Does it generate a link without `(url)`?